search/create_datastore_and_search.ipynb

```diff
@@ -0,0 +1,3120 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #15.            parent=f"projects/{PROJECT_ID}/locations/{LOCATION}/collections/default_collection",
</pre><p>You can use this method to create this path instead of hardcoding it</p><p> </p><p><a href="https://cloud.google.com/python/docs/reference/discoveryengine/latest/google.cloud.discoveryengine_v1alpha.services.data_store_service.DataStoreServiceClient#google_cloud_discoveryengine_v1alpha_services_data_store_service_DataStoreServiceClient_collection_path" target="_blank">https://cloud.google.com/python/docs/reference/discoveryengine/latest/google.cloud.discoveryengine_v1alpha.services.data_store_service.DataStoreServiceClient#google_cloud_discoveryengine_v1alpha_services_data_store_service_DataStoreServiceClient_collection_path</a></p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/271/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>